src/ralph/backends/data/base.py

```diff
@@ -132,15 +133,15 @@ def write(  # noqa: PLR0913
             operation_type = self.default_operation_type
 
         if operation_type in self.unsupported_operation_types:
-            msg = f"{operation_type.value.capitalize()} operation_type is not allowed."
+            msg = f"{operation_type.value.capitalize()} operation_type is not allowed"
             self.logger.error(msg)
             raise BackendParameterException(msg)
 
         data = iter(data)
         try:
             first_record = next(data)
         except StopIteration:
-            self.logger.info("Data Iterator is empty; skipping write to target.")
+            self.logger.info("Data Iterator is empty; skipping write to target")
```

Comment:
Wondering if this should be `warning` ?

Reply:
IMO info level is sufficient, it's the caller fault and no action has been made